src/hw9/ArrayValueCalculator.java

```diff
@@ -3,25 +3,25 @@
 public class ArrayValueCalculator{
 
 
-        public static int doCalc(String[][] arr) throws ArraySizeException, ArrayDataException {
-            // Check the size of the array
-            if (arr.length != 4 || arr[0].length != 4) {
-                throw new ArraySizeException("The input array must be of size 4x4");
-            }
+    public static int doCalc ( String[][] arr ) throws ArraySizeException, ArrayDataException{
```

Comment:
The same here, formatting suffers :( 
replace `public static int doCalc ( String[][] arr )` with `public static int doCalc(String[][] arr)`